py/client/pydeephaven/session.py

```diff
@@ -240,8 +244,17 @@ def _fetch_fields(self):
     def _connect(self):
         with self._r_lock:
             try:
-                self._flight_client = paflight.connect(location=(self.host, self.port), middleware=[
```

Comment:
Note the `connect` call has been deprecated: https://github.com/apache/arrow/blob/c62ce6b179f67ff57fa1571b01f356739727c0c9/python/pyarrow/_flight.pyx#L1354